org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ConverterTestSetup.java

```diff
@@ -354,6 +340,14 @@ public void setUpSuite() throws Exception {
 		}
 	}
 
+	protected IJavaProject setUpJavaProject(final String projectName, String compliance) throws CoreException, IOException {
+		this.currentProject =  setUpJavaProject(projectName, compliance, false);
+		if(CompilerOptions.getFirstSupportedJavaVersion().equals(compliance)) {
+			this.currentProject.setOption(JavaCore.COMPILER_PB_RAW_TYPE_REFERENCE, JavaCore.IGNORE);
+		}
+		return this.currentProject;
+	}
+
```

Comment:
@srikanth-sankaran  @jarthana  as you may expect I'm not a fan of suppressing compile problems at large during tests. OTOH, the majority of converter tests don't care about problems (some do, though).

I'll put my worries at rest, iff at least one of you approves this change :)

Reply:
As I've explained above, not suppressing this warning would need manual adaptation of multiple converter tests written in pre-generics era. So far I don't see how suppressing this option should be a problem for tests that didn't care about generics before. Also note, the option was ignored in JDT tests ~25 times before, so seem to be a common practice. I've added the question to TODO whether we should use that in `ConverterTestSetup`.